src/utils/weatherDataMapper.test.ts

```diff
@@ -257,6 +257,7 @@ describe("Weather Data Mapper", () => {
             length: 10000,
             width: 150,
             surface: "CONCRETE",
```

Comment:
The API returns "A" or "C" (and possibly other letters). It doesn't return the word "CONCRETE". The test data should just show "C" or "A".